LICENSE.md

```diff
@@ -1,22 +1 @@
-There are ongoing discussions with the Colorado Department of State
-about which Open Source license is most appropriate for this system.
-
-They have agreed to license it under a variant of the GPLv3, but the
-final decision about which variant it will be licensed under is still
-under discussion.
-
-In particular, should it be licensed under
-[AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) or
-[GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html)? Please see the
-FSF document, ["Why the Affero
-GPL"](https://www.gnu.org/licenses/why-affero-gpl.en.html), for a
-relevant discussion.
-
-If you, the reader, have input on this decision, please add a comment
-to [Issue #452](https://github.com/FreeAndFair/ColoradoRLA/issues/452).
-
-Thus, as of now, this system is not licensed for any use whatsoever
-beyond its owner, the Colorado Department of State.
-
-
-
+The Colorado RLA Tool is Copyright (C) 2017 the Colorado Department of State, and is licensed under the [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html)  with a classpath exception.
```

Comment:
I think we should follow normal practice and simply include the full AGPLv3. That keeps things self-contained, makes searches work better, etc.

Reply:
OK, will do.